app/models/feedback_submission.rb

```diff
@@ -1,18 +1,32 @@
-class FeedbackSubmission < ActiveRecord::Base
-  validates :body, presence: true
-  validate :validate_email
+class FeedbackSubmission
+  include NonPersistedModel
 
-  before_validation :strip_email_address, on: :create
+  attribute :body, String
+  attribute :email_address, String
```

Comment:
I would prefer providing a an API endpoint to validate the email, or even a separate microservice.